
[2016-06-24 10:45:07] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: params' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php:287
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php(287): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 287, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(95): Nova\Routing\Controller->callAction('index', Array)
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(64): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Welcome), Object(Nova\Routing\Route), 'index')
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(923): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'index')
#4 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(106): call_user_func_array(Object(Closure), Array)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(987): Nova\Routing\Route->run(Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-06-24 10:45:08] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:45:10] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: params' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php:287
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php(287): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 287, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(95): Nova\Routing\Controller->callAction('session', Array)
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(64): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Demo), Object(Nova\Routing\Route), 'session')
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(923): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'session')
#4 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(106): call_user_func_array(Object(Closure), Array)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(987): Nova\Routing\Route->run(Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-06-24 10:45:10] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: params' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php:287
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php(287): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 287, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(95): Nova\Routing\Controller->callAction('index', Array)
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(64): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Settings\Controllers\Admin\Settings), Object(Nova\Routing\Route), 'index')
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(923): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Set...', 'index')
#4 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(106): call_user_func_array(Object(Closure), Array)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(987): Nova\Routing\Route->run(Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-06-24 10:45:11] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: params' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php:287
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php(287): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 287, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(95): Nova\Routing\Controller->callAction('index', Array)
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(64): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Users\Controllers\Admin\Roles), Object(Nova\Routing\Route), 'index')
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(923): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#4 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(106): call_user_func_array(Object(Closure), Array)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(987): Nova\Routing\Route->run(Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-06-24 10:45:11] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: params' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php:287
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Controller.php(287): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 287, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(95): Nova\Routing\Controller->callAction('index', Array)
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/ControllerDispatcher.php(64): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Users\Controllers\Admin\Users), Object(Nova\Routing\Route), 'index')
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(923): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#4 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(106): call_user_func_array(Object(Closure), Array)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(987): Nova\Routing\Route->run(Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-06-24 10:45:14] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:21] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:21] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:21] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:21] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:21] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:27] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:27] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:27] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:27] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:27] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:28] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:28] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:28] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:28] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 10:46:28] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1008): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(977): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(956): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 11:05:30] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1023): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(992): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(971): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 11:18:33] testing.ERROR: exception 'Symfony\Component\Console\Exception\InvalidArgumentException' with message 'The helper "table" is not defined.' in /var/www/sites/novatesting/vendor/symfony/console/Helper/HelperSet.php:79
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Console/RoutesCommand.php(80): Symfony\Component\Console\Helper\HelperSet->get('table')
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Console/Command.php(103): Nova\Foundation\Console\RoutesCommand->fire()
#2 /var/www/sites/novatesting/vendor/symfony/console/Command/Command.php(256): Nova\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Console/Command.php(91): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/sites/novatesting/vendor/symfony/console/Application.php(815): Nova\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/sites/novatesting/vendor/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Nova\Foundation\Console\RoutesCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/sites/novatesting/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /var/www/sites/novatesting/forge(39): Symfony\Component\Console\Application->run()
#8 {main} [] []
[2016-06-24 11:34:41] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1024): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(993): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(972): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
[2016-06-24 11:38:25] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: regex' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php:144
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(144): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 144, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(118): Nova\Routing\Route->compileRoute()
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(220): Nova\Routing\Route->matches(Object(Nova\Http\Request), true)
#3 [internal function]: Nova\Routing\RouteCollection->Nova\Routing\{closure}('/', Object(Nova\Routing\Route))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/helpers/functions.php(346): call_user_func(Object(Closure), '/', Object(Nova\Routing\Route))
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(221): array_first(Array, Object(Closure))
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(131): Nova\Routing\RouteCollection->check(Array, Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1024): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(993): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(972): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#12 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#18 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#19 {main} [] []
[2016-06-24 11:38:26] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: regex' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php:144
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(144): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 144, Array)
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Route.php(118): Nova\Routing\Route->compileRoute()
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(220): Nova\Routing\Route->matches(Object(Nova\Http\Request), true)
#3 [internal function]: Nova\Routing\RouteCollection->Nova\Routing\{closure}('/', Object(Nova\Routing\Route))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/helpers/functions.php(346): call_user_func(Object(Closure), '/', Object(Nova\Routing\Route))
#5 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(221): array_first(Array, Object(Closure))
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php(131): Nova\Routing\RouteCollection->check(Array, Object(Nova\Http\Request))
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1024): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(993): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#9 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(972): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#12 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#18 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#19 {main} [] []
[2016-06-24 11:40:42] testing.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/RouteCollection.php:146
Stack trace:
#0 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(1024): Nova\Routing\RouteCollection->match(Object(Nova\Http\Request))
#1 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(993): Nova\Routing\Router->findRoute(Object(Nova\Http\Request))
#2 /var/www/sites/novatesting/vendor/nova-framework/system/src/Routing/Router.php(972): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#3 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(691): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#4 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(675): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#5 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#6 /var/www/sites/novatesting/vendor/nova-framework/system/src/Session/Middleware.php(72): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#7 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#8 /var/www/sites/novatesting/vendor/nova-framework/system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#9 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#10 /var/www/sites/novatesting/vendor/nova-framework/system/src/Foundation/Application.php(571): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#12 {main} [] []
